contents/handbook/company/team/joe-martin.md

```diff
@@ -8,30 +8,54 @@ showTitle: true
-
-# What I value
-- Compassion and inclusivity. Anything that puts people first, really. 
-- A 'no bad ideas' attitude. I don't think there's any such thing as a stupid question, even though I ask a lot of them.
-- Vulnerability, especially in leaders. It's healthy and builds trust. I'm happy to be vulnerable with you too.    
-- Honest feedback. I am literally always interested in hearing how I can improve.
-- A hands-on attitude. I don't think I've ever said 'That's not my job'.
+- I can help with literally anything involving writing, editing or proof-reading.
+- I enjoy helping others. You can ask me anything. There are no stupid questions.  
+
+### How to work with me
+- I default to listening. You may need to ask for my thoughts if you want them.
+- I have a very hands-on attitude and will never say 'That's not my job'. 
+- I may parrot ('_It sounds like..._') to ensure I understand. If I'm wrong, tell me. 
+- I'm truly uncomfortable with being in the limelight. I prefer private thank yous.
+- I value compassion and inclusivity above everything else. I try to put people first. 
+- Ad-hoc meetings are fine, but weekends are sacred. I generally work 9-6 GMT.
+
+## Current areas of focus
+### Execution - Q1 2022
+- [ ] **Community:** Organize the first PostHog community meet-up, with a good speaker. Retro the event and roll changes into a second event. Continue to attend and report back on other community events and meet-ups.
+- [ ] **Case studies:** Secure an enterprise-level case study. Enhance the Customers section of the site and make plans to better leverage this content.
+- [ ] **Content:** Support the Content team in creating a decent pipeline of good content that continues to deliver.
+- [ ] **Partnerships:** Find further partnership opportunities to grow the community.
+- [ ] **Internal:** Continue to advocate for less technical team members and users. Become more ambitious when it comes to shipping.
+
+### Personal - Q1 2022
+- [ ] Don't lose (another, again) mission in Pandemic Legacy: Season Zero.
+- [ ] See all of my nieces at least once. And my nephew.
+- [ ] Don't miss a Donut chat.
+
+## Feedback
+In our last [360 feedback session](/handbook/people/feedback#full-team-feedback-sessions-1), I received feedback on the following points. 
+
+- I am seen as an effective and flexible executor, planner, writer and storyteller.
+- I am seen as empathetic and extremely supportive of colleagues, especially new colleagues or those with less experience than me. 
+- I am seen as brave when giving feedback and stepping on toes. I default to transparency.
+- Work I do, such as [case studies](/customers) is seen as valuable and showing a lot of insight.
+- I am seen as over-cautious and too detail-orientated when planning new work. 
+- I am seen as over-communicative in social discussions and on the topic of diversity. 
```

Comment:
All looks great to me, just this point - I had a different take on from the feedback session on diversity specifically. My point wasn't that you were over-communicative but more challenging you to turn that communication/discussion into further action. 

I think 'over-communicative' makes it sounds like you talk about diversity too much, which I certainly wouldn't want to discourage you from doing!

(If you are synthesizing other people's feedback here though and I've missed the point, please disregard - I'm biased towards my own feedback to you on this point.)

Reply:
Yeah, this is probably a fault of me trying to be too succint. Will update.